src/components/NavBar.vue

```diff
@@ -185,6 +190,11 @@ export default {
         onSearchTextChange(searchText) {
             this.searchText = searchText;
         },
+        async fetchAuthConfig() {
+            this.fetchJson(this.authApiUrl() + "/config").then(config => {
+                this.registrationDisabled = config?.registrationDisabled === true;
```

Comment:
I don't really like this because the config will now be fetched twice on each lagd reload, since the footer component does the same in order to receive the URLs in the footer.

Reply:
Me too, but I thought this was okay since we're fetching the configuration of the authentication API. We need to make a universal way of loading it for the oidc PR too.

Reply:
I was also thinking of loading the config and caching it somewhere so it does not have to be loaded twice if authApi = api but i thought this might be out-of-scope for this PR